ambry-clustermap/src/main/java/com.github.ambry.clustermap/ClusterMapManager.java

```diff
@@ -45,6 +48,24 @@
 
   private Logger logger = LoggerFactory.getLogger(getClass());
 
+  // Comparators for comparing unallocated capacity in datanodes/disks
+  private final Comparator<DataNode> dataNodeCapacityComparator = new Comparator<DataNode>() {
+    @Override
+    public int compare(DataNode o1, DataNode o2) {
+      long o1Capacity = getUnallocatedRawCapacityInBytes(o1);
```

Comment:
minor: can probably write a function for these three lines in the outer class and reuse it.